src/lib/y2network/config_reader/autoinst.rb

```diff
@@ -43,7 +43,7 @@ def initialize(section)
       def config
         interfaces = find_interfaces
         routing = section.routing ? AutoinstRouting.new(section.routing).config : nil
-        Y2Network::Config.new(interfaces: interfaces, routing: routing, source: :autoyast)
```

Comment:
The `source: :sysconfig` instead of `:autoinst` is a workaround we should discuss. The writer used is based on the source, and that is not the case for AutoYaST. That is, the target writer should be decided based on the current network backend and not based on the source from where the config was read. So basically I used :sysconfig to use its writer.

Reply:
makes sense for me to just have backend value in Config.